### DISCUSSION TITLE
Token length

### DISCUSSION THREAD

**Discussion by Alstruit** [iptv-org/community#31] [Q&A] [answered]
What is the acceptable time limit of a valid tokenied m3u8? Some tokenized links provide a time limit before the generated m3u expires. It could be in a range from a few short minutes to hours or even a few days. Should a expire label be included as well?

**Comment by BellezaEmporium** [comment, marked as answer]
Most of the time they are configured to last for an hour approximatively.

There's some rare cases in which they can last for a day, or 4/8/12 hours, but rarely over that.

**Comment by TKList** [comment]
And once in a while they work 24/7 if you remove the token.

**Reply by BellezaEmporium** [reply]
Doesn't happen all the time, but yes, some do not control the token's presence inside the link and simply works lol
